src/aggregation_rules/aligned_time_rule.rs

```diff
@@ -72,16 +72,34 @@ where
 
 #[cfg(test)]
 mod tests {
+    use trade_aggregation_derive::Candle;
+
     use super::*;
+
     use crate::{
-        aggregate_all_trades, load_trades_from_csv, plot::OhlcCandle, GenericAggregator, Trade, M15,
+        aggregate_all_trades,
+        candle_components::{
+            CandleComponent, CandleComponentUpdate, Close, High, Low, NumTrades, Open, Volume,
+        },
+        load_trades_from_csv, GenericAggregator, ModularCandle, TimestampResolution, Trade, M1,
+        M15,
     };
 
+    #[derive(Default, Debug, Clone, Candle)]
+    struct MyCandle {
+        open: Open,
+        close: Close,
+        high: High,
+        low: Low,
+        num_trades: NumTrades<u32>,
+        volume: Volume,
```

Comment:
```suggestion
        open: Open,
        close: Close,
        num_trades: NumTrades<u32>,
        volume: Volume,
```
Looks like `high` and `low` are not used in the test.